plugins/processors/converter/converter.go

```diff
@@ -12,20 +12,23 @@ import (
 
 	"github.com/influxdata/telegraf"
 	"github.com/influxdata/telegraf/filter"
+	"github.com/influxdata/telegraf/internal"
 	"github.com/influxdata/telegraf/plugins/processors"
 )
 
 //go:embed sample.conf
 var sampleConfig string
 
 type Conversion struct {
-	Measurement []string `toml:"measurement"`
-	Tag         []string `toml:"tag"`
-	String      []string `toml:"string"`
-	Integer     []string `toml:"integer"`
-	Unsigned    []string `toml:"unsigned"`
-	Boolean     []string `toml:"boolean"`
-	Float       []string `toml:"float"`
+	Measurement     []string `toml:"measurement"`
+	Tag             []string `toml:"tag"`
+	String          []string `toml:"string"`
+	Integer         []string `toml:"integer"`
+	Unsigned        []string `toml:"unsigned"`
+	Boolean         []string `toml:"boolean"`
+	Float           []string `toml:"float"`
+	Timestamp       string   `toml:"timestamp"`
```

Comment:
I would still use an array for uniformity and as its also done like this for the `measurement` name.

```suggestion
	Timestamp       []string `toml:"timestamp"`
```